src/modules/show_player_in_hltv.rs

```diff
@@ -0,0 +1,47 @@
+//! `Show player model in HLTV spectate`
+
+use super::Module;
+use crate::hooks::{bxt, engine};
+use crate::utils::*;
+
+pub struct ShowPlayerInHltv;
+impl Module for ShowPlayerInHltv {
+    fn name(&self) -> &'static str {
+        "Show player model in HLTV spectate"
+    }
+
+    fn description(&self) -> &'static str {
+        "\
+When people do cam path, they usually want to see the player in the demo. This means the demo will be recorded in third person. \
+The only way to enter third person and free camera is to do `dem_forcehltv 1` and then use `viewdemo` command. \
+For a brief moment in free spectate in `viewdemo`, the player model will show up. \
+However, after cycling through all spectating options once, the player model disappears.
+
+This is the fix for that.
+
+Alternatively, with this fix, it is possible to use `playdemo` with `thirdperson` to have the player model show up to just record. \
+Free spectate is not available in `playdemo` nonetheless."
+    }
+
+    fn is_enabled(&self, marker: MainThreadMarker) -> bool {
+        // BXT has the same functionality.
+        !bxt::BXT_TAS_LOAD_SCRIPT_FROM_STRING.is_set(marker)
+            && engine::cls_demos.is_set(marker)
+            && engine::ClientDLL_IsThirdPerson.is_set(marker)
+            && engine::CL_IsSpectateOnly.is_set(marker)
+            && engine::pmove.is_set(marker)
+            && unsafe { engine::find_cmd(marker, "dem_forcehltv") }.is_some()
```

Comment:
Hmm, I don't like this logic involving some engine state in `is_enabled()`. Perhaps should find this somewhere similar to how we find pointers?

Reply:
These HLTV and demo codes are inside demoplayer.so (or at the very least, not inside engine code). I don't see any references to static HLTV variables inside engine. The reason why there is cvar check in the first place is because BXT does it. I suspect it is to avoid crashing or unintended behaviors. @SmileyAG may know why it is needed.

Anyways, if it doesn't look like what you want, it is possible to store a static variable caching the result of this find_cmd during run time to check that `dem_forcehltv` actually exists.